src/components/AnnualReport/AGMDate.vue

```diff
@@ -128,24 +128,19 @@ export default class AgmDate extends Mixins(DateMixin) {
   }
 
   /** New AGM Date (from a resumed draft). */
-  @Prop({ default: null })
-  readonly newAgmDate: string
+  @Prop({ default: null }) readonly newAgmDate!: string
```

Comment:
The `!` was required in one of the files to fix a lint error.

Ref: https://blog.logrocket.com/understanding-exclamation-mark-typescript

I applied it to all props for consistency.